tools/lsp/ui/views/header-view.slint

```diff
@@ -17,35 +18,49 @@ export component HeaderView {
         background: Palette.alternate-background;
 
         content-layer := HorizontalBox {
-            alignment: end;
+            HeaderText {
+                text: @tr("Preview");
+                horizontal-stretch: 1;
```

Comment:
Ah yes, I had the same issue when I tried to prototype this. In an ideal world we could just use `alignment: end;` and have a `align-self: start;` on just this one.

Reply:
You can also add a Rectangle as a spacer item.

Iirc we discussed a long time ago of we should have a Spacer but decided that Rectangle was good enough